src/Storage/Query/Handler/PagingHandler.php

```diff
@@ -22,20 +22,20 @@ public function __invoke(ContentQueryParser $contentQuery)
 
         foreach ($contentQuery->getContentTypes() as $contenttype) {
             //Find out if we are searching or just doing a simple query
-            if ($searchParam = $contentQuery->getParameter('filter')) {
+            if ($contentQuery->hasParameter('filter')) {
```

Comment:
Hmm … I've been tossing that over in my head actually, as we're going to hit more of this as we progress though v3's life cycle.

3.2.15 is 100% an option, but it seems a touch hostile to some potentially, or maybe I am over thinking it. I mean, upgrading from 3.2.x to 3.2.15 is a no-brainer, but if someone hits a bug in the extension and needs to update, they might invalidate UAT requirements in the process… "Hello, overthinking!"